### PR TITLE
Formatting and copy paste fixes

### PR DESCRIPTION
**Pull request by aleschneck**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (colourgarden) [inline .vscode/launch.json:4]:
```suggestion
```

Comments not valid in JSON files. If these need to be there then they need to be in key/value format.

Comment (colourgarden) [inline .vscode/launch.json]: Not valid JSON but I guess the VSCode parser accepts them so probably ok to leave.

File: dist/index.js [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
...on(t,e){this.merges.filterIntersects(t).forEach((function(t){return e(t)}))}},{key:"hideRowsOrCols",value:function(){var t=this.rows,e=this.cols,n=this.selector,r=ut(n.size(),2),i=r[0],o=r[1],a=n.range,l=a.sri,c=a.sci,s=a.eri,u=a.eci;if(i===t.len)for(var f=c;f<=u;f+=1)e.setHide(f,!0);else if(o===e.len)for(var h=l;h<=s;h+=1)t.setHide(h,!0)}},{key:"unhideRowsOrCols",value:function(t,e){this["".concat(t,"s")].unhide(e)}},{key:"rowEach",value:function(t,e,n){for(var r=0,i=this.rows,o=this.exceptRowSet,a=at(o),l=0,c=0;c<a.length;c+=1)a[c]<t&&(l+=1);for(var s=t+l;s<=e+l;s+=1)if(o.has(s))l+=1;else{var u=i.getHeight(s);if(u>0&&(n(s,r,u),(r+=u)>this.viewHeight()))break}}},{key:"colEach",value:function(t,e,n){for(var r=0,i=this.cols,o=t;o<=e;o+=1){var a=i.getWidth(o);if(a>0&&(n(o,r,a),(r+=a)>this.viewWidth()))break}}},{key:"defaultStyle",value:function(){return this.settings.style}},{key:"addStyle",value:function(t){for(var e=this.styles,n=0;n<e.length;n+=1){var r=e[n];if(F.equals(r,t))return n}return e.push(t),e.length-1}},{key:"changeData",value:function(t){this.history.add(this.getData()),t(),this.change(this.getData())}},{key:"setData",value:function(t){var e=this;return Object.keys(t).f
```
Comment (colourgarden) [inline dist/index.js]: @aleschneck Could we exclude `dist` from the repo and then build a release for the project via a GitHub Action when a commit is made to `master`?

File: src/component/sheet.js [hunk 3]
```diff
@@ -809,7 +816,6 @@ function sheetInitEvents() {
         case 86:
           // ctrl + v
           // => paste
-          clearClipboard.call(this);
           // evt.preventDefault();
```
Comment (colourgarden) [inline src/component/sheet.js]: @aleschneck Can these lines be removed?

File: src/core/data_proxy.js [hunk 4]
```diff
@@ -132,7 +132,7 @@ function copyPaste(srcCellRange, dstCellRange, what, autofill = false) {
   const { rows, merges } = this;
   // delete dest merge
   if (what === 'all' || what === 'format') {
-    rows.deleteCells(dstCellRange, what);
+    // rows.deleteCells(dstCellRange, what);
```
Comment (colourgarden) [inline src/core/data_proxy.js]:
```suggestion
```

**Review by colourgarden — APPROVED** [review]
Approving to not delay but take a look at the comments.

File: dist/index.js (hunk 2 above)
Comment (aleschneck) [reply dist/index.js]: I'm down, will set it up for the next PR

**Pull request merged**